eventdiscovery-sdk/src/main/res/layout/schedjoules_view_event_details_description.xml

```diff
@@ -7,7 +7,7 @@
             android:layout_height="wrap_content"
             android:layout_width="match_parent"
             android:paddingLeft="@dimen/schedjoules_activity_horizontal_margin"
```

Comment:
Shouldn't the description have a left padding of `schedjoules_activity_horizontal_margin_with_icon_padding_inset` to align it with the icon padding instead of the icon boundaries?